src/components/userService.ts

```diff
@@ -0,0 +1,47 @@
+import { Memento } from 'vscode';
+
+export class GlobalStateTokenStorage {
```

Comment:
this class should implement `IGlobalStateTokenStorage` explicitly

Reply:
addressed